### PR TITLE
Create future.is-an.app.js

### PR DESCRIPTION
**Pull request by Huseyin-Cinar**
### Link to Website

Link: 

---

- [ ] Toggle this checkbox to re-run the stargazer checking

### CODE REVIEW

File: domains/future.is-an.app.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+addSubDomain({
+  description: 'domain for my website',
+  domain: 'is-an.app',
+  subdomain: 'future',
+  owner: {
+    email: 'gslicocuk8080@yaani.com',
+  },
+  record: {
+    CNAME: 'github.com/The-Future-Software-Company',
```
Comment (tarampampam) [inline domains/future.is-an.app.js:9]: Wrong cname record

**Review by tarampampam — CHANGES_REQUESTED** [review]

File: domains/future.is-an.app.js (hunk 1 above)
Comment (Huseyin-Cinar) [reply domains/future.is-an.app.js:9]: What is wrong?

Comment (tarampampam) [reply domains/future.is-an.app.js:9]: https://en.wikipedia.org/wiki/List_of_DNS_record_types

File: domains/future.is-an.app.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+addSubDomain({
+  description: 'domain for my website',
+  domain: 'is-an.app',
+  subdomain: 'future',
+  owner: {
+    email: 'future.software.company@gmail.com',
+  },
+  record: {
+    CNAME: 'github.com/the-future-software-company',
```
Comment (tarampampam) [inline domains/future.is-an.app.js:9]:
```suggestion
    CNAME: 'the-future-software-company.github.io',
```

File: domains/future.is-an.app.js (hunk 1 above)
Comment (tarampampam) [reply domains/future.is-an.app.js]: https://github.com/tarampampam/free-domains/pull/134#discussion_r1118403016

File: domains/future.is-an.app.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+addSubDomain({
+  description: 'domain for my website',
+  domain: 'is-an.app',
+  subdomain: 'future',
+  owner: {
+    email: 'future.software.company@gmail.com',
+  },
+  record: {
+    CNAME: 'The-Future-Software-Company.github.io',
```
Comment (tarampampam) [inline domains/future.is-an.app.js:9]: Change to lowercase, please

File: domains/future.is-an.app.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+addSubDomain({
+  description: 'domain for my website',
```
Comment (tarampampam) [inline domains/future.is-an.app.js:2]: Provide a more details, please

**Review by tarampampam — CHANGES_REQUESTED** [review]

File: domains/future.is-an.app.js (hunk 3 above)
Comment (Huseyin-Cinar) [reply domains/future.is-an.app.js]: I did it.

File: domains/future.is-an.app.js (hunk 4 above)
Comment (Huseyin-Cinar) [reply domains/future.is-an.app.js]: I did it.

**Comment by Huseyin-Cinar** [comment]
thanks.

**Comment by Huseyin-Cinar** [comment]
image

@tarampampam

**Comment by tarampampam** [comment]
Without knowledge about the expected result and what you already did - I can't help you, sorry 😞

**Pull request merged**